src/components/Header.js

```diff
@@ -26,6 +24,9 @@ export default function(store){
       filters() {
         return store.state.filters;
       },
+      zipcode() {
```

Comment:
Just realized that whatever is referencing zipcode could also just get it out of `filters.zipcode`, but either way